tox.ini

```diff
@@ -74,4 +74,7 @@ commands =
     {toxinidir}/scripts/run_regression.sh
 deps =
     unittest2
+#   Use a development checkout of oauth2client until a release is made
+#   which fixes https://github.com/google/oauth2client/issues/125
+    -egit+https://github.com/google/oauth2client.git#egg=oauth2client
```

Comment:
`-e` makes it an actual checkout, rather than using a temporary checkout to create a then-installed egg.  It allows us us hack on the package directly (in `.tox/regression3/src/oauth2client`) while getting `regression3` to pass.